cloud/jenkins/pxc_operator_aws_openshift-4.groovy

```diff
@@ -33,7 +33,7 @@ compute:
   platform:
     aws:
       type: m5.2xlarge
-  replicas: 3
+  replicas: 1
```

Comment:
Do we have test runs with these changes?

Reply:
yes, https://cloud.cd.percona.com/view/PG/job/pgo-operator-aws-openshift-4/

Reply:
ok, but they have failures, let's make them run so we can see the whole run